src/NUnitFramework/framework/nunit.framework.csproj

```diff
@@ -1,56 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\Common.props" />
   <PropertyGroup>
-    <AssemblyName>nunit.framework</AssemblyName>
-    <RootNamespace>NUnit.Framework</RootNamespace>
     <TargetFrameworks>net20;net35;net40;net45;netstandard1.6</TargetFrameworks>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <OutputPath>..\..\..\bin\$(Configuration)\</OutputPath>
-    <DocumentationFile>..\..\..\bin\$(Configuration)\$(TargetFramework)\nunit.framework.xml</DocumentationFile>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
```

Comment:
Are the defaults for `ErrorReport` and `WarningLevel` "prompt" and 4? (same comment for the other projects).

Reply:
Yes with the new csproj format. Woth the old format only WarningLevel was redundant.